.github/workflows/release.yaml

```diff
@@ -74,6 +74,12 @@ jobs:
           echo 'release_notes<<EOF' >> $GITHUB_ENV
           cat docs/release_notes/${{ steps.prep.outputs.VERSION }}.md >> $GITHUB_ENV
           echo 'EOF' >> $GITHUB_ENV
+      - name: Build Profiles Userdocs
+        uses: peter-evans/repository-dispatch@0e8ca8c8a5ca5e28d25af2c27fc5aa40f16cffca
+        with:
+          token: ${{ secrets.WEAVEWORKS_PCTL_USERDOC_DISPATCH_TOKEN }}
+          event-type: pctl-release
+          repository: weaveworks/profiles-userdocs
```

Comment:
what does this do?

Reply:
Triggers a [repository_dispatch](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#repository_dispatch) event.

Reply:
what does that mean? We have some API webhook running in the `profiles-userdoc` repo that recieves this request?

Reply:
Read the docs, Jake. :D

Reply:
It's all Github. There is nothing running anywhere. It's a trigger type for actions. :) Respectively in profiles-userdocs there is a trigger like this:

```yaml
on:
  repository_dispatch:
    types: [pctl-release, profiles-release]
```

Which will react to the triggered event with the right keywords.